File: FATT_backend/WebAPI/Models/Dish.cs
namespace WebAPI.Models
{
    public class Dish
    {
        public long Id { get; set; }
        public string? Name { get; set; } 
        public string? Preptime { get; set; } 

        public string? Ingredients { get; set; } 
        public string? NutritionalValue { get; set; } 
        public string? Recipe { get; set; } 
        public string? PicturePath { get; set; }
        public List<Meal> Meals { get; set; } = new();
    }
}

File: FATT_frontend/FATT_NEXT/fatt/interfaces/Dish.ts
import type { MealJustNameDto } from "./Meal";

export type Dish = {
  id: number;
  name: string;
  prepTime: string;
  ingredients: string;
  nutritionalValue: string;
  recipe: string;
  picturePath: string;
};

export type Dishs = {
  data: Dish[];
};

export type DishNoIdDto = {
  name: string;
  prepTime: string;
  ingredients: string;
  nutritionalValue: string;
  recipe: string;
  picturePath: string;
};

export type DishJustNameDto = {
  id?: number;
  name: string;
};
